src/components/missions/MissionEntry.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
```

Comment:
to import components from react-bootstrap you can regroup them to save more line in code like that .i.e : 
```JS
import { Button, Badge } from 'react-bootstrap';
```